gems/aws-sdk-core/spec/seahorse/client/async_base_spec.rb

```diff
@@ -0,0 +1,86 @@
+require_relative '../../spec_helper'
+
+module Seahorse
+  module Client
+    describe AsyncBase do
+      if RUBY_VERSION >= '2.3'
```

Comment:
There needs to be a counter to this, where the RUBY_VERSION is < 2.3 and we raise an appropriate error.